components/post_emoji/post_emoji.tsx

```diff
@@ -1,11 +1,24 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
+import React, {CSSProperties} from 'react';
 import PropTypes from 'prop-types';
 
-export default class PostEmoji extends React.PureComponent {
-    static propTypes = {
+interface PostEmojiProps {
+    name: string;
+    imageUrl: string;
+}
+declare module 'react' {
+    interface HTMLAttributes<T> extends AriaAttributes, DOMAttributes<T> {
+        alt?: string;
+        className?: string;
+        title?: string;
+        style?: CSSProperties;
+    }
+}
+
+export default class PostEmoji extends React.PureComponent<PostEmojiProps, {}> {
+    public static propTypes = {
```

Comment:
These are no longer needed now that we're using TypeScript.

Reply:
@devinbinnie taken them out.